### PR TITLE
Istanbul colors in grunt output

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: tasks/intern.js [hunk 1]
```diff
@@ -104,6 +104,9 @@ module.exports = function (grunt) {
 			}
 		});
 
+		// force colored output for istanbul report
```
Comment (csnover) [inline tasks/intern.js]: Please use the environment variable FORCE_COLOR instead

File: tests/selftest.intern.js [hunk 2]
```diff
@@ -16,6 +16,10 @@ define({
 
 	maxConcurrency: 2,
 	tunnel: 'BrowserStackTunnel',
+	tunnelOptions: {
```
Comment (csnover) [inline tests/selftest.intern.js]: This information is always provided via command-line and shouldn’t be put in the test file

**Comment by csnover** [comment]
Thanks for your patch!

**Comment by stdavis** [comment]
You are welcome! Thanks for the great project.

On Wed, Jun 24, 2015 at 6:18 PM, Colin Snover <notifications@github.com>
wrote:

> Thanks for your patch!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theintern/intern/pull/423#issuecomment-115052247>.
>



-- 
*Scott Davis*
Utah AGRC <http://gis.utah.gov/developer/>
stdavis@utah.gov
@ScottAGRC <https://twitter.com/ScottAGRC>

**Pull request closed without merging**